### PR TITLE
Adds new wasteplanet cave type

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Concrete Caves. They're caves but cement.

dreamseeker_GF0CtB249n
dreamseeker_3Ko7axdwIy

## Why It's Good For The Game
planetary variety.

## Changelog

:cl:
add: Wasteplanets now will generate concrete filled caves. 
/:cl:

### CODE REVIEW

**Comment by meemofcourse** [comment]
can we have like. a picture

**Comment by Erikafox** [comment]
few secs please

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 1]
```diff
@@ -66,9 +66,9 @@
 		BIOME_COLD_CAVE = list(
 			BIOME_LOWEST_HUMIDITY = /datum/biome/cave/waste,
 			BIOME_LOW_HUMIDITY = /datum/biome/cave/waste/rad,
-			BIOME_MEDIUM_HUMIDITY = /datum/biome/cave/waste,
-			BIOME_HIGH_HUMIDITY = /datum/biome/cave/waste/rad,
-			BIOME_HIGHEST_HUMIDITY = /datum/biome/cave/waste
+			BIOME_MEDIUM_HUMIDITY = /datum/biome/cave/waste/conc,
+			BIOME_HIGH_HUMIDITY = /datum/biome/cave/waste/conc,
+			BIOME_HIGHEST_HUMIDITY = /datum/biome/cave/waste/conc
```
Comment (FalloutFalcon) [inline code/datums/mapgen/planetary/WasteGenerator.dm:71]: I dont really feel like this needs to be 3/5 of the cave.. I would have it be (waste waste rad conc conc)

Comment (FalloutFalcon) [reply code/datums/mapgen/planetary/WasteGenerator.dm:71]: actually im dumb nvm this is ok

**Pull request merged**